### PR TITLE
#20896 Testing IntegrityCjecker

### LINKED ISSUE #20896: Create integration test for Integrity Checker
Create test for IntegrityUtil.completeCheckIntegrity method

### PR DESCRIPTION
**Pull request by freddyucv**
Testing IntegrityChecker

### CODE REVIEW

**Review by jdotcms — APPROVED** [review]

File: dotCMS/src/integration-test/java/com/dotcms/integritycheckers/IntegrityUtilTest.java [hunk 1]
```diff
@@ -0,0 +1,463 @@
...
+        IntegrityUtil.completeCheckIntegrity(endPointID);
+
+        DotConnect dc = new DotConnect();
+        dc.setSQL("select * from folders_ir where endpoint_id = ?");
+        dc.addObject(endPointID);
+        final List<Map<String, Object>> results = dc.loadObjectResults();
+
+        assertEquals(1, results.size());
+
+        assertEquals(folder.getIdentifier(), results.get(0).get("local_inode"));
```
Comment (dsilvam) [inline dotCMS/src/integration-test/java/com/dotcms/integritycheckers/IntegrityUtilTest.java:109]: Is this ok? looks like it should be `folder.getInode`

File: dotCMS/src/integration-test/java/com/dotcms/integritycheckers/IntegrityUtilTest.java [hunk 2]
```diff
@@ -0,0 +1,463 @@
...
+        assertEquals(folder.getInode(), results.get(0).get("local_identifier"));
```
Comment (dsilvam) [inline dotCMS/src/integration-test/java/com/dotcms/integritycheckers/IntegrityUtilTest.java:110]: Same here. Looks like it should be 'folder.getIdentifier()'

File: dotCMS/src/integration-test/java/com/dotcms/integritycheckers/IntegrityUtilTest.java [hunk 3]
```diff
@@ -0,0 +1,463 @@
...
+        assertEquals(workingContentlet.getInode(), results.get(0).get("local_working_inode"));
+        assertEquals(newWorkingInode_1, results.get(0).get("remote_working_inode"));
+        assertEquals(liveContentlet.getIdentifier(), results.get(0).get("local_identifier"));
+        assertEquals(newIdentifier_1, results.get(0).get("remote_identifier"));
+        assertEquals(endPointID, results.get(0).get("endpoint_id"));
+        assertEquals(language.getId(), results.get(0).get("language_id"));
+    }
+
+    @Test
+    public void pageFileWithConflicts() throws Exception {
```
Comment (dsilvam) [inline dotCMS/src/integration-test/java/com/dotcms/integritycheckers/IntegrityUtilTest.java:202]: Missing javadoc

File: dotCMS/src/integration-test/java/com/dotcms/integritycheckers/IntegrityUtilTest.java (hunk 1 above)
Comment (freddyucv) [reply dotCMS/src/integration-test/java/com/dotcms/integritycheckers/IntegrityUtilTest.java]: done https://github.com/dotCMS/core/pull/21743/commits/06e0b7fd1a20f8d94d51bbb98ee2f99262306ff1#diff-955f4199431261a5a18c844f03e72d5d53a98da73e62f16374c76aaa41a611fdL109

File: dotCMS/src/integration-test/java/com/dotcms/integritycheckers/IntegrityUtilTest.java (hunk 2 above)
Comment (freddyucv) [reply dotCMS/src/integration-test/java/com/dotcms/integritycheckers/IntegrityUtilTest.java]: done https://github.com/dotCMS/core/pull/21743/commits/06e0b7fd1a20f8d94d51bbb98ee2f99262306ff1#diff-955f4199431261a5a18c844f03e72d5d53a98da73e62f16374c76aaa41a611fdL109

File: dotCMS/src/integration-test/java/com/dotcms/integritycheckers/IntegrityUtilTest.java (hunk 3 above)
Comment (freddyucv) [reply dotCMS/src/integration-test/java/com/dotcms/integritycheckers/IntegrityUtilTest.java:209]: done https://github.com/dotCMS/core/pull/21743/commits/06e0b7fd1a20f8d94d51bbb98ee2f99262306ff1#diff-955f4199431261a5a18c844f03e72d5d53a98da73e62f16374c76aaa41a611fdR202

**Review by dsilvam — APPROVED** [review]

**Pull request merged**